internal/cli/atlas/deployments/search/indexes/create.go

```diff
@@ -282,7 +280,7 @@ func CreateBuilder() *cobra.Command {
 
 			return opts.PreRunE(
 				opts.InitOutput(w, createTemplate),
-				opts.InitStore(opts.PodmanClient),
+				opts.InitStore(cmd.Context()),
```

Comment:
[nit] having `opts.InitStore` and `opts.initStore` together is not perfect, ideally in a followup we should fix it everywhere